cardano_node_tests/cluster_scripts/babbage/genesis.alonzo.spec.json

```diff
@@ -1,372 +1,365 @@
 {
-    "maxBlockBodySize": 90112,
-    "lovelacePerUTxOWord": 34482,
-    "executionPrices": {
```

Comment:
There are two ways to represent rational numbers, so the changed version is fine too